hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -1025,6 +1030,11 @@ public Action cancelAction(final long actionId) {
         }
     }
 
+    @Override
+    public void updateActionExternalRef(final long actionId, final String externalRef) {
+        actionRepository.updateExternalRef(actionId, externalRef);
```

Comment:
we should check that the action exists in db first

Reply:
Same as reason as [this comment](https://github.com/bsinno/hawkbit/pull/10#discussion_r278477564)